packages/web-components/src/components/ic-text-field/ic-text-field.tsx

```diff
@@ -45,6 +45,35 @@ import {
 let inputIds = 0;
 const MUTABLE_ATTRIBUTES = [...IC_INHERITED_ARIA, "title"];
 
+const ignoredKeyboardCharacters = [
```

Comment:
Could you please move this into the [constants file](https://github.com/mi6/ic-ui-kit/blob/v3.0.0/develop/packages/web-components/src/utils/constants.ts)? It might be something we want to use in other components down the line